PSSM_score_Peptide/main.cpp

```diff
@@ -179,13 +179,9 @@ void getFileNamesFromArgv(int argc, char *argv[], string & PSSM_FileName, string
 		else if (string(argv[i]) == "-pssm_cutoffs") CutofsPerPSSM_FileName = string(argv[i + 1]);
 		else if (string(argv[i]) == "-seq") Seq_FASTA_FileName = string(argv[i + 1]);
 		else if (string(argv[i]) == "-out") Hits_Out_FileName = string(argv[i + 1]);
-		else if (string(argv[i]) == "-NrandPSSM") numberOfRandomPSSM=size_t(atoi(argv[i + 1]));
-		else if (string (argv[i]) == "-useFactor"){
-			if (string(argv[i + 1]) == "True"){
-			use_factor = true;
-			}
-		}
-
+		else if (string(argv[i]) == "-NrandPSSM") numberOfRandomPSSM = size_t(atoi(argv[i + 1]));
+		else if (string (argv[i]) == "-useFactor") useFactor = (atoi(argv[i + 1]));
```

Comment:
No need for value, the flag means setting the variable to true.
Set the default to false before at start of function